float64 x
float64 y
float64 heading
float64 roll
float64 pitch
float64 dr_x
float64 dr_y
float64 speed
geometry_msgs/Quaternion orientation